modules/ROOT/pages/api-groups-landing-page.adoc

```diff
@@ -3,12 +3,12 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-API Groups enables you to bundle a collection of APIs that you use based on your working environment, such as Sandbox and Design. You use these bundles to easily share information across applications, services, and systems, including legacy systems. 
+API Groups enables you to bundle a collection of APIs that you use based on your working environment, such as Sandbox and Design. Because each API Group can include several API instances, and each instance can be included in multiple API Group solutions, package customization options are limitless. 
```

Comment:
```suggestion
API Groups enables you to bundle a collection of APIs that you use based on your working environment, such as Sandbox and Design. Because each API Group instance can include several API instances, and each instance can be included in multiple API Group instance solutions, package customization options are limitless. 
```

Reply:
Done! Thanks!